src/AppBundle/Resources/views/Report/ProfDeputyCostsEstimate/_answers.html.twig

```diff
@@ -16,10 +16,10 @@
         {# How did you charge for your costs? #}
         <tr class="behat-region-how-charged">
                 <td>
-                    {{ 'howCharged.form.profDeputyCostsEstimateHowCharged.label' | trans(transOptions) }}
+                    {{ 'howCharged.form.profDeputyCostsEstimateHowCharged.label' | trans }}
                 </td>
                 <td>
-                    {{ howCharged(report.profDeputyCostsEstimateHowCharged) }}
+                    {{ ('howCharged.form.options.' ~  report.profDeputyCostsEstimateHowCharged) | trans }}
```

Comment:
This replaces the use of the `howCharged` TwigExtension that I wrote a couple of weeks back. I actually think yours is a more simple solution which I must have overlooked for some reason. I'll carry out a separate task to remove the extension from the code (-: